projects/kubernetes-sigs/image-builder/packer/ami/additional_files/logging.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
This is a new file, in order to have it copied to the ami, can you add an entry [here](https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/kubernetes-sigs/image-builder/packer/ami/ansible_extra_vars.yaml#L3)?

Reply:
added the entry, thanks.